### PR TITLE
Added history feature and fixed dev env to start develop new feature

### PR DESCRIPTION
**Pull request by CVarisco**
- In this PR you can find like a new Terminal the history of the commands.
- Also, some fixing with the dev env to work on the dev env.

I'm here for every question or editing.
Thanks man!

### CODE REVIEW

File: components/Terminal.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ import './Terminal.css';
 console.oldLog = console['log']; // eslint-disable-line no-console, dot-notation
 
 function handleLogging(method, addToOutput) {
-  console[method] = (...args) => { // eslint-disable-line no-console
+  console[method] = (...args) => {
+    // eslint-disable-line no-console
```
Comment (jcgertig) [inline components/Terminal.js]: This does not work has to be on the same line. If you really don't like it on the same line you can add a line between 9 and 10 and put `// eslint-disable-next-line`

File: components/Terminal.js [hunk 2]
```diff
@@ -153,21 +193,30 @@ class Terminal extends Component {
...
-                  <span className="terminal-prompt" style={prompt}>{this.state.prompt}</span>
+                  <span className="terminal-prompt" style={prompt}>
+                    {this.state.prompt}
+                  </span>
                   <input
                     className="terminal-main-input"
                     style={inputStyles}
                     type="text"
-                    ref={com => (this.com = com)}
+                    ref={com => this.com = com}
```
Comment (jcgertig) [inline components/Terminal.js]: The linter will not like this.

Comment (CVarisco) [inline components/Terminal.js]: Running `npm run lint` I don't get the any suggestion about this.
I will change the to older version of this line 👍

File: components/Terminal.js (hunk 1 above)
Comment (CVarisco) [inline components/Terminal.js]: Hi @jcgertig 
I'm using prettier-eslint that on save lint the files.
However, I can change the lines of code 🙂

**Comment by CVarisco** [comment]
Is it ok now ? 🙂 
Thanks 🌮

**Comment by nitin42** [comment]
Just give me some time. I will review the code later because I'm working on a new feature. Don't worry I will merge this and thanks for the PR 😃

**Comment by nitin42** [comment]
Also I posted this on Reddit and some comments are so depressing.

**Comment by CVarisco** [comment]
@nitin42 depressing ? for what ?

**Comment by nitin42** [comment]
Nothing they just anticipated much more. I've just started working and I've a ton of ideas to improve this component further.

**Comment by nitin42** [comment]
Btwn thanks for being patient 😃

**Comment by jcgertig** [comment]
Yeah I usually wait a bit to post to reddit

**Comment by nitin42** [comment]
Next time I'll take care of this !

File: package.json [hunk 3]
```diff
@@ -38,16 +38,16 @@
     "webpack-dev-server": "^2.5.1"
   },
   "scripts": {
-    "start": "NODE_ENV=development ./node_modules/.bin/webpack-dev-server --config ./webpack.config.js --open --progress --colors --content-base starter/ --hot",
+    "start": "NODE_ENV=development ./node_modules/.bin/webpack-dev-server --config ./webpack.config.js --open --progress --colors --content-base starter/",
     "build:production": "NODE_ENV=production ./node_modules/.bin/webpack --config ./webpack/webpack.config.js --display-max-modules 0",
     "prepublish": "npm run build:production",
     "prebuild": "rm -rf ./build",
-    "lint": "./node_modules/.bin/eslint components/",
+    "lint": "./node_modules/.bin/eslint components/"
```
Comment (nitin42) [inline package.json]: Already made this change!

File: starter/App.js [hunk 4]
```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import { render } from 'react-dom';
-// Bundle generated with npm run build:production 
-import Terminal from '../build/terminal.js';
+// Bundle generated with npm run build:production
+import Terminal from '../components/Terminal';
```
Comment (nitin42) [inline starter/App.js]: We need this. You can uncomment it but don't replace this.

File: starter/App.js [hunk 5]
```diff
@@ -1,10 +1,8 @@
...
 
-const App = () => (
-  <Terminal msg="Hello World. My name is Nitin Tulswani" />
-);
+const App = () => <Terminal msg="Hello World. My name is Nitin Tulswani" />;
```
Comment (nitin42) [inline starter/App.js]: Thanks !

File: webpack.config.js [hunk 6]
```diff
@@ -9,24 +9,23 @@ module.exports = {
...
+      },
+    ],
   },
   target: 'web',
   plugins: [
     new webpack.DefinePlugin({
       'process.env': {
-        NODE_ENV: JSON.stringify('development')
-      }
+        NODE_ENV: JSON.stringify('development'),
```
Comment (nitin42) [inline webpack.config.js]: Again, I've made the changes already!

**Comment by jcgertig** [comment]
I really want this working on command updates has been a pain with out it

**Comment by nitin42** [comment]
Hey @CVarisco can you do a `git pull` and commit your changes again (for adding history feature) ? I've merged `139ea95` btwn. 

History feature looks really great and I'm excited. Thank you!

**Comment by jcgertig** [comment]
@CVarisco you will need to do a new pr with these changes because a lot has changed

File: starter/App.js (hunk 4 above)
Comment (CVarisco) [inline starter/App.js]: The hot module doesn't work before your changes with this path.
Now it works 🎉

Comment (nitin42) [inline starter/App.js]: Yeah!

**Comment by CVarisco** [comment]
I created a new [PR](https://github.com/nitin42/terminal-in-react/pull/14) here with the feature in the new terminal structure.
I hope that is ok now 👍

**Pull request merged**